File: IssueTracker.Common/Models/Audit.cs
using IssueTracker.Common.Models.Base;

namespace IssueTracker.Common.Models
{
	public class Audit : NameModel
	{
		public string Property { get; set; }
		public string OldValue { get; set; }
		public string NewValue { get; set; }
	}
}

File: old-data/models/audit.js
var mongoose = require("mongoose");
var objectId = mongoose.Schema.Types.ObjectId;

var schema = mongoose.Schema({
    name: String,
    isDeleted: { type: Boolean, default: false },
    property: String,
    oldValue: String,
    newValue: String,
    
    issueAudit: { type: objectId, ref: "issue-audit" }
});

module.exports = require("bluebird").promisifyAll(mongoose.model("audit", schema));